.github/workflows/main.yml

```diff
@@ -77,8 +78,7 @@ jobs:
       - run: cargo test -F $FEATURE -F profiler
       - run: cargo test -F $FEATURE --tests -- --ignored
       - run: cargo test -F $FEATURE --manifest-path examples/Cargo.toml
-      - run: cargo test --manifest-path bonsai/examples/governance/Cargo.toml
-        if: matrix.device == 'cpu'
```

Comment:
Was this intentional? This is probably what's causing all the build failures. You need to specify `-F $FEATURE` if you want this to work on GPUs. But I also wonder what we gain by running this test on GPU.

Reply:
I think we should also considering making the workspace `bonsai/examples` rather than `bonsai/examples/governance`

Reply:
> I think we should also considering making the workspace bonsai/examples rather than bonsai/examples/governance

The main goal of the showcase is to demonstrate what a more-or-less full project looks like. If pull the workspace declarations out into `bonsai/examples` it makes it harder to copy this as a self-contained starting point/example. Maybe we can consider this again when we add a second Bonsai example? By then we will hopefully have also extracted the `relay` crate, which is not actually specific to this example.

Reply:
I'm starting to think maybe we can extend the `risczero` tool to turn examples into standalone projects.